test/Common.h

```diff
@@ -83,4 +83,6 @@ struct CommonOptions: boost::noncopyable
 	static std::unique_ptr<CommonOptions const> m_singleton;
 };
 
+bool isValidSemanticTestPath(boost::filesystem::path const& _testPath);
```

Comment:
I'd add documentation here:

```suggestion
// Returns true if the path is within `test/libsolidity/semanticTests` and it does not start with an underscore.
bool isValidSemanticTestPath(boost::filesystem::path const& _testPath);
```

Reply:
Yep :) https://github.com/ethereum/solidity/pull/11110#discussion_r616971108